projects/tinkerbell/hook/Makefile

```diff
@@ -9,39 +9,39 @@ REPO_OWNER=tinkerbell
 BASE_IMAGE_NAME?=eks-distro-minimal-base
 DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
 
-BOOTKIT_IMAGE_COMPONENT=tinkerbell/hook-bootkit
-TINK_DOCKER_IMAGE_COMPONENT=tinkerbell/hook-docker
+HOOK_BOOTKIT_IMAGE_COMPONENT=tinkerbell/hook-bootkit
```

Comment:
all these renames are intentional?  Looking over the changes here the resulting image names and what not are all the same so I think you should be good there, in terms of not needing new ecr repos and what not, but double check that.

Reply:
yeah, intentional. I found i needed them updated to get all the tooling to build correctly.

Reply:
also, we wont need any ecr repos for these images as they are intermediate images (they live in the build process side car registry) that get only consumed by the Hook in order to build itself.